.circleci/config.yml

```diff
@@ -28,6 +30,16 @@ jobs:
             pip install -r requirements.txt -r tests/requirements.txt
             pip install wheel twine
 
+      - run:
+          name: Install appropriate dimod version
+          command: |
+            . env/bin/activate
+            if [[ "<< parameters.dimod-version >>" ]]; then
+              pip install 'dimod<< parameters.dimod-version >>'
```

Comment:
this may mess with whatever numpy version you installed from `requirements.txt`. Probably doesn't matter but just in case.

Reply:
Shouldn't that dimod install pull the appropriate numpy?

Reply:
Yup, just that you already installed one in requirements, so if for whatever reason you wanted to pin that version in the future you might be surprised it gets changed in CI.

Reply:
Oh, that's what you mean. Yeah, that's perfectly fine. I use numpy range only here anyway (so it can be determined by dimod).

Reply:
As I said, don't think it matters :shrug: